server/src/subscriber/subscriber.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, Res } from "@nestjs/common";
+import { ConfigService } from "../config/config.service";
+import { Client } from "@sendgrid/client";
+const crypto = require('crypto');
```

Comment:
Change this to `import crypto from 'crypto'`. You'll get a type error which can be resolved with `yarn add @types/node`